### PR TITLE
gh-1674 Add note about x.y.z-0 versions considered to be releases only

### PR DESCRIPTION
**Pull request by veod32**

### CODE REVIEW

File: doc/dev_guide/release_management.rst [hunk 1]
```diff
@@ -115,6 +115,14 @@ channel (`tarantool_news <https://t.me/tarantool_news>`_).
 We also publish nightly builds, and use the fourth slot in the version
 identifier to designate the nightly build number.
 
+.. important::
+
+   Only a version with ``0`` in the fourth slot, namely ``X.Y.Z-0-<hash>`` is
```
Comment (Totktonada) [inline doc/dev_guide/release_management.rst:120]: `X.Y.Z-0-g<hash>` to be precise. 'g' is 'git' I guess.

File: doc/dev_guide/release_management.rst [hunk 2]
```diff
@@ -115,6 +115,14 @@ channel (`tarantool_news <https://t.me/tarantool_news>`_).
...
+   considered to be a *release*. Further nightly builds in the same branch
+   although published are **not** releases. For example, ``2.5.1-0-<hash>`` is
```
Comment (Totktonada) [inline doc/dev_guide/release_management.rst:122]: > Further nightly builds in the same branch although published are **not** releases.

This is nitpicking, but I would try to reword it to make simpler. Like so, maybe:

> Packages that are published with non-zero fourth slot are nightly builds, not releases.

**Review by Totktonada — APPROVED** [review]
LGTM in general, but, please, consider a few tiny comments above.

No need to re-review with me.

File: doc/dev_guide/release_management.rst (hunk 1 above)
Comment (veod32) [reply doc/dev_guide/release_management.rst:120]: corrected

File: doc/dev_guide/release_management.rst (hunk 2 above)
Comment (veod32) [reply doc/dev_guide/release_management.rst:122]: re-writed
